Scripts/Features/Windows-OptionalFeatures.ps1

```
@@ -52,10 +52,44 @@ function Test-WindowsOptionalFeatureEnabled {
 
     try {
         $feature = Get-WindowsOptionalFeature -Online -FeatureName $FeatureName -ErrorAction Stop
+        return $feature.State -eq 'Enabled'
     }
     catch {
         return $false
     }
+}
+
+function Set-WindowsHibernate {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Set-WindowsHibernate' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)